include/RAJA/policy/MultiPolicy.hpp

```diff
@@ -100,8 +98,8 @@ using policy::multi::MultiPolicy;
 
 namespace detail
 {
-template <size_t... Indices, typename... Policies, typename Selector>
-auto make_multi_policy(VarOps::index_sequence<Indices...>,
+template <camp::idx_t... Indices, typename... Policies, typename Selector>
```

Comment:
Thanks for this, definitely needed to be done, and it should be faster this way on most compilers now as well. :+1: